java/src/main/java/ai/rapids/cudf/WindowOptions.java

```diff
@@ -50,13 +57,18 @@ public static Builder builder(){
 
   ColumnVector getFollowingCol() { return this.followingCol; }
 
+  int getTimestampColumnIndex() { return this.timestampColumnIndex; }
+
+  FrameType getFrameType() { return frameType; }
```

Comment:
Now that these can be set we need to validate in the existing windowing code that they are set to what it would expect or throw an exception.  Otherwise people may get confused about what is and isn't supported.

Reply:
I've now modified `ColumnVector#rollingWindow()` to verify that the `FrameType` is `ROWS`, before proceeding. Added a test for it as well.